xarray/__init__.py

```diff
@@ -52,6 +52,12 @@
     # Disable minimum version checks on downstream libraries.
     __version__ = "999"
 
+try:
+    from datatree import DataTree  # noqa
```

Comment:
```suggestion
    from datatree import DataTree, register_datatree_accessors  # noqa: F401
```
I think this is why the docs build is failing. Also, not sure if the error code still works with `ruff`?

Reply:
I've changed this to 
```python
from datatree import DataTree, register_datatree_accessor, open_datatree  # noqa
```
now though